src/dvclive/live.py

```diff
@@ -882,7 +883,8 @@ def make_dvcyaml(self):
 
     @catch_and_warn(DvcException, logger)
     def post_to_studio(self, event: Literal["start", "data", "done"]):
-        post_to_studio(self, event)
+        thread = threading.Thread(target=post_to_studio, args=(self, event))
```

Comment:
A few questions:

- do we still need `@catch_and_warn(DvcException, logger)` above?
- do we start a new thread per each request now? sound expensive and can for example exhaust OS if post_to_studio is running faster then we process them
- can we use a single thread (and a queue for example)?

Reply:
why not run start and done through the same queue? it's better a bit since start will initialize the queue - makes it a bit more thread-safer